### ISSUE TITLE
[User Feedback] Custom `extra` context does not display in UI with User Feedback API method

### ISSUE THREAD

**Issue by Fwang36** [getsentry/sentry#73194]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

tested in SDK `@sentry/nextjs 8.11.0` and `@sentry/node 8.11.0`

1. [Follow docs here](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-api) on how to apply scope modifications to user feedback event.
2. See that the `extra` key does not appear in User Feedback.

```
    const eventId = Sentry.captureMessage("User Feedback");
    

    const userFeedback = {
        name: "John Doe",
        email: "john@doe.com",
        message: "I really like your App, thanks!",
        associatedEventId: eventId,
      };
    
      Sentry.captureFeedback(
          userFeedback,
          {
            captureContext: {
              tags: { testingKey: "someKeyvalue" },
              extra: { testingExtra: "someExtraValue" },
            },
          },
        );
```

The outgoing feedback event network request includes the extra key -

<img width="336" alt="Screenshot 2024-06-24 at 10 01 54 AM" src="https://github.com/getsentry/sentry/assets/96670625/b30339e8-70db-4e4d-a785-d47c84ea848b">

The UI only shows the added tags but not the added Extra data -

<img width="779" alt="Screenshot 2024-06-24 at 10 02 27 AM" src="https://github.com/getsentry/sentry/assets/96670625/c67bacd4-f7cf-4d04-8aaa-a7f9c9194709">

[Link to the feedback event in the screnshots in my org.](https://francisorg.sentry.io/feedback/?feedbackSlug=nextjs14%3A5529978405&project=-1&statsPeriod=14d)

### Expected Result

Extra data should show in the UI

### Actual Result

Extra data does not show.

### Product Area

User Feedback

**Comment by bruno-garcia** [comment]
Thanks for raising! We don't support `extra`'s just yet. But we'll keep track of it through this ticket

**Comment by bruno-garcia** [comment]
For context, the SDK part is unblocked to pass `extra` as you've observed, but Sentry isn't yet showing that data:

https://github.com/getsentry/sentry-javascript/issues/10916#issuecomment-2134664701

**Comment by Fwang36** [comment]
Ah I see, thanks!

Is there a timeline or plans to implement support for it currently?

If not yet, should we maybe remove the code block [example here in the docs](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-api) for now?  
<img width="500" alt="Screenshot 2024-06-24 at 3 32 31 PM" src="https://github.com/getsentry/sentry/assets/96670625/96bba1be-a58f-42d9-955a-32055e019aa0">

**Comment by bruno-garcia** [comment]
We don't have a timeline for that yet. Removing the note from the doc until then is a good call, thanks

**Comment by aliu39** [comment]
@bruno-garcia do we still want to support this?

**Comment by max-degterev** [comment]
Second parameter of Sentry.captureFeedback seems a no-op at the moment, is that the same for everyone? Attachments are not passed along at all

**Comment by bruno-garcia** [comment]
@max-degterev please open a new ticket. The second parameter takes more than just `extra` so if nothing shows up it's unrelated to this issue. Please state if you're on SaaS or self hosted, and which version if the latter.
